olive/passes/pytorch/gptq.py

```diff
@@ -0,0 +1,161 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+from typing import Any, Dict
+
+import torch
+
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import PyTorchModelHandler
+from olive.model.handler.onnx import ONNXModelHandler
+from olive.passes import Pass
+from olive.passes.onnx.common import get_external_data_config
+from olive.passes.pass_config import PassConfigParam
+from olive.passes.pytorch.gptq_utils import QuantLinearORT
+
+logger = logging.getLogger(__name__)
+
+ROUNDTRIP_CHECK = False
```

Comment:
Curious that where this flag is used?

Reply:
Oh, It's a debug line and will be delete